Source/JavaScriptCore/heap/IsoSubspace.h

```diff
@@ -100,7 +100,7 @@ ALWAYS_INLINE Allocator IsoSubspace::allocatorFor(size_t size, AllocatorForMode)
 
 } // namespace GCClient
 
-#define ISO_SUBSPACE_INIT(heap, heapCellType, type) ("IsoSpace " #type, (heap), (heapCellType), sizeof(type), type::usePreciseAllocationsOnly, type::numberOfLowerTierPreciseCells)
+#define ISO_SUBSPACE_INIT(heap, heapCellType, type) (ASCIILiteral::fromLiteralUnsafe("IsoSpace " #type), (heap), (heapCellType), sizeof(type), type::usePreciseAllocationsOnly, type::numberOfLowerTierPreciseCells)
```

Comment:
I’m surprised there’s no way to do this. Like `"IsoSpace " #type _s` maybe?

Reply:
There are definitely ways to do this with `_s`. We've done it in other places. I need to find an example.

Reply:
This would be an example:
```
Source/JavaScriptCore/runtime/SymbolConstructor.cpp:putDirectWithoutTransition(vm, Identifier::fromString(vm, #name ""_s), Symbol::create(vm, static_cast<SymbolImpl&>(*vm.propertyNames->name##Symbol.impl())), P
```